superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -566,13 +566,15 @@ class SqlEditor extends React.PureComponent {
             <Checkbox checked={this.state.autocompleteEnabled} />{' '}
             {t('Autocomplete')}
           </Button>{' '}
-          <TemplateParamsEditor
-            language="json"
-            onChange={params => {
-              this.props.actions.queryEditorSetTemplateParams(qe, params);
-            }}
-            code={qe.templateParams}
-          />
+          {isFeatureEnabled(FeatureFlag.ENABLE_TEMPLATE_PROCESSING) && (
```

Comment:
We are showing "Parameters" in SQL Lab even though the functionality is now behind a feature flag in the backend. I turned it off here if the functionality is not enabled.

cc: @hughhhh